app/models/dashboard.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Summary of recent activity across the whole app
+class Dashboard
+  def recent_uploads_by_provider
+    Rails.cache.fetch(:recent_uploads_by_provider, expires_in: 1.hour) do
+      Upload.recent.where('created_at > ?', 30.days.ago).group_by(&:organization)
+    end
+  end
+
+  def job_status_groups_by_provider
+    Rails.cache.fetch(:job_status_groups_by_provider, expires_in: 1.hour) do
+      Organization.providers.index_with { |org| org.default_stream.job_tracker_status_groups }
+    end
```

Comment:
Does the 1 hour cache time here mean that e.g. jobs that complete within the 1 hour window won't show as completed until 1 hour has elapsed? Not sure how the caching works.

Reply:
I think this query could either not be cached (it's not that slow) or it could be cached very briefly, maybe a minute or something.

Reply:
Yeah, with stuff like this I worry that people will be looking to this dashboard for up-to-the-minute information, and then be confused or start to lose trust in it when they find that it doesn't actually represent the system state sometimes (and by "people" I mean me).

Reply:
@thatbudakguy @corylown I had this same concern about caching vs. up to date info -- we also have some info in a different tab that is based on the Statistics job, so I worry about having 2-3 multiple different time scales for data on this page.

Reply:
I agree that it's probably okay if the page is slow initially. The problem in my mind is the paging presented with the uploads list -- it's unbearably slow to wait for the data to load for each page. I wonder if this points to the need for a different design, where the uploads appear on their own page where the tabbed data isn't being loaded.

Reply:
See comment in https://github.com/pod4lib/aggregator/issues/672